src/components/posts/PostItem.js

```diff
@@ -134,10 +143,10 @@ const PostItem = ({ post, userId, setPostResponse, setPost, setUser, user, setPr
   };
 
   return (
-    <li className="post-item">
-      <div className="post-wrap">
-        <div className="post-header-wrap">
-          <div className="post-profile-wrap">
+    <li className={`post-item ${post.isPostOfTheWeek ? 'post-of-the-week' : null}`}>
```

Comment:
It might be easier to read if we split this out of the JSX at define a variable for the className in the component. 

something like:
```js
let liClasses = 'post-item'
if (post.isPostOfTheWeek) {
   liClasses += ' post-of-the-week'
}
```

Reply:
Done!